mitosheet/mitosheet/tests/streamlit/test_runnable_analysis.py

```diff
@@ -445,7 +445,7 @@ def function(vari\abl"e_name{}):
         'name': 'vari\ abl"e_name{}'
     },
 ]
-    analysis = RunnableAnalysis('', None, special_characters_fn, special_characters_metadata)
+    analysis = RunnableAnalysis('', None, None, special_characters_fn, special_characters_metadata)
```

Comment:
... This is a breaking change to the RunnableAnalysis interface. See if we can make this optional instead

Reply:
Oh nvm, it doesn't make sense to include in the RunnableAnalysis, so just removing it